README.md

```diff
@@ -146,9 +146,9 @@ An initial agenda will be posted to the [shared Google docs][shared-meeting-note
 
 Meeting recordings are uploaded to official etcd [YouTube channel].
 
-Get calendar invitation by joining [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev) mailing group.
+Get calendar invitation by joining [etcd-dev](https://groups.google.com/g/etcd-dev) mailing group.
 
-Join Hangouts Meet: [meet.google.com/umg-nrxn-qvs](https://meet.google.com/umg-nrxn-qvs)
+Join CNCF-funded Zoom channel: [zoom.us/my/cncfetcdproject](https://zoom.us/my/cncfetcdproject)
 
 Join by phone: +1 405-792-0633‬ PIN: ‪299 906‬#
```

Comment:
Suggest we remove this phone number also, zoom provides audio dial in instructions through the normal flow when joining the meeting.

Reply:
ah, ok. Didn't know that it's related to the virtual meeting room.

Reply:
removed